### PR TITLE
CFY-7142. Fix command line too long

### PR DESCRIPTION
**Pull request by jcollado**
In this PR, the code that sends the agent installation script to a windows server is updated to split the script into chunks.

The reason to split the script is that to the script is included as part of a powershell command, so its size is restricted by the maximum command line length supported in windows to avoid getting a command line too long error.

### CODE REVIEW

File: cloudify_agent/installer/runners/winrm_runner.py [hunk 1]
```diff
@@ -470,12 +483,50 @@ def close(self):
...
+
+    :param contents:
+        The contents of a file that exceeds the maximum command line length in
+        windows.
+    :type content: str
+    :returns: The same content in chunks that won't exceed the limit
+    :rtype: list[str]
+
+    """
+    # Split content into chunks to avoid command line too long error
```
Comment (mcouthon) [inline cloudify_agent/installer/runners/winrm_runner.py]: This seems to be repeating the text in the docstring

File: cloudify_agent/installer/runners/winrm_runner.py [hunk 2]
```diff
@@ -470,12 +483,50 @@ def close(self):
...
+    # maximum allowed commmand line length should be 2047 in old windows:
+    # https://support.microsoft.com/en-us/help/830473/command-prompt-cmd--exe-command-line-string-limitation
+    max_size = 2000
+    separator = '\r\n'
+
+    def join_lines(lines, line):
```
Comment (mcouthon) [inline cloudify_agent/installer/runners/winrm_runner.py]: This is pretty neat! But I think I'd like a unittest for this particular function - just to see that we get exactly the results we want. And maybe play around with different strings and escape characters inside the split string

File: cloudify_agent/installer/runners/winrm_runner.py (hunk 1 above)
Comment (jcollado) [inline cloudify_agent/installer/runners/winrm_runner.py]: Fixed in ee019f5

File: cloudify_agent/installer/runners/winrm_runner.py (hunk 2 above)
Comment (jcollado) [inline cloudify_agent/installer/runners/winrm_runner.py]: Fixed in 4bb35d6

**Pull request merged**